Formula/mockserver.rb

```diff
@@ -1,8 +1,8 @@
 class Mockserver < Formula
   desc "Mock HTTP server and proxy"
   homepage "https://www.mock-server.com/"
-  url "https://search.maven.org/remotecontent?filepath=org/mock-server/mockserver-netty/5.11.2/mockserver-netty-5.11.2-brew-tar.tar"
-  sha256 "1758bb80c3e5cd250b55757a53105db7b03ff1b05d2dfca501ce5795feff8756"
+  url "https://oss.sonatype.org/content/repositories/releases/org/mock-server/mockserver-netty/5.13.0/mockserver-netty-5.13.0-brew-tar.tar"
+  sha256 "b6106b856608aa8f2b6829b330e16d4ae5f7954881b7732496de204d68143d84"
```

Comment:
Why did the source change?

Reply:
The original url returns a redirect to the new url so I assume the `brew bump-formula-pr` tool used the redirected url and not the original url.

Reply:
The behaviour of these urls has not changed only the way the `brew bump-formula-pr` interprets them.

Reply:
For me it redirects to https://repo1.maven.org/maven2/org/mock-server/mockserver-netty/5.13.0/mockserver-netty-5.13.0-brew-tar.tar . What command did you run?

Reply:
Yes sorry now I release I had changed that URL a while back but the last release to homebrew failed for a separate reason.  The command I ran was:
```bash
brew bump-formula-pr --strict mockserver --url=https://oss.sonatype.org/content/repositories/releases/org/mock-server/mockserver-netty/5.13.0/mockserver-netty-5.13.0-brew-tar.tar
```
I now remember the reason this was changed was because the new URL downloads the file directly instead of doing a look up first, given we know the direct location there is no reason to first lookup.  More importantly the new URL uses sonatype directly which has the file immediately after a release where as the old URL has a variable delay before the file was present.  Typically is takes two hours (due to a background syncing process) before the old url would work making the release process require a variable delay. 

Does that make sense?

Reply:
variable 2 hours +- 1 hour is a problem for an automation pipeline.

What is the problem with the URL changing?

Reply:
Well, the source of software changing is a rather big change which, without a possibly to validate the change, could mean an attempted hijacking of the package to distribute... anything really.

Reply:
Yes I totally agree that makes sense and I'm not doubting your effort to ensure that a hacker doesn't slip in a hacked binary.

Do I need to re-issue the PR or given this has switched from a public well known Maven endpoint to a public well known Sonatype (who own Maven central) endpoint is that enough in this case to allow the change in URL?